app/assets/stylesheets/application.css

```diff
@@ -8,6 +8,6 @@
  * You're free to add application-wide styles to this file and they'll appear at the top of the
  * compiled file, but it's generally better to create a new file per style scope.
  *
+ *= require ./spree_admin
  *= require_self
- *= require_tree .
```

Comment:
this requires all stylesheet files but doesn't fulfill the order of the dependencies among them. `layout.css` fails because can't find the variable `$color-1`.